helm/azure-ad-pod-identity-app/templates/nmi-daemonset.yaml

```diff
@@ -26,26 +28,32 @@ spec:
       {{- if .Values.rbac.enabled }}
       serviceAccountName: {{ template "aad-pod-identity.nmi.fullname" . }}
       {{- end }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
       {{- if .Values.nmi.priorityClassName }}
       priorityClassName: {{ .Values.nmi.priorityClassName | quote }}
       {{- end }}
-      hostNetwork: true
-      dnsPolicy: ClusterFirstWithHostNet
       volumes:
-      - hostPath:
+      - name: iptableslock
+        hostPath:
           path: /run/xtables.lock
-          type: FileOrCreate
-        name: iptableslock
+          type: FileOrCreate 
+      - name: kubelet-config
+        hostPath:
+          path: {{ .Values.kubelet.path }}
       containers:
       - name: nmi
         image: "{{ .Values.image.registry }}/{{ .Values.nmi.image.name }}:{{ .Values.nmi.image.tag }}"
         imagePullPolicy: {{ .Values.image.imagePullPolicy }}
         args:
           - "--host-ip=$(HOST_IP)"
           - "--node=$(NODE_NAME)"
+          - --http-probe-port={{ .Values.nmi.probePort | default 8080 }}
+          - --operation-mode={{ .Values.operationMode }}
+          {{- if eq .Values.operationMode "managed" }}
+          - --forceNamespaced
```

Comment:
Why is this added when we run managed?

Reply:
It's what's used in capz and in the aad upstream chart:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/2e25580076af010974c0beb0fedd9d01155d36ce/config/default/aad-pod-identity-deployment.yaml#L110

Reply:
Ohh okay - so "managed" means "like it is in CAPI"

Reply:
Technically, standard means the mic component will assign then identities and managed that something else will. In our case, it should be capi yes